### PR TITLE
Moving direct Allas read from GDAL to own tutorial page

### PR DESCRIPTION
**Pull request by samumantha**
* new tutorial page: direct read from Allas and other object storage
* removed direct read from Allas stuff from GDAL page and replaced with link to new page
* also added link to geoconda and gis-r page

Please **squash** and merge :)

### CODE REVIEW

File: docs/support/tutorials/gis/direct_allas.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+The GDAL command line tool can only __read__ files from external location, but not write. If you want to write files to Allas, you can still save them first to the supercomputer, virtual machine or own computer and move them to Allas later. In R and Python it is possible also to __write__ to Allas directly from script. We have tested successfully:
+
+ * [Python](../../../apps/geoconda.md): gdal, geopandas, fiona and rasterio. [Example codes on CSCs github repository](https://github.com/csc-training/geocomputing/blob/master/python/allas/working_with_allas_from_Python_S3.py). 
+ * [R](../../../apps/r-env-for-gis.md): sf, raster. [Example codes on CSCs github repository](https://github.com/csc-training/geocomputing/blob/master/R/allas/working_with_allas_from_R_S3.R). 
+ * [QGIS](../../../apps/qgis.md)
+
+ 
+Reading data directly from an external service, like Allas or other object storage is slower than reading from local disks, for example reading a ~500 Mb files from your scratch directory on Puhti takes ~1 second, reading the same file from Allas takes ~10 seconds. In many cases, these seconds are negligible compared to the full duration of an analysis.
```
Comment (EetuHuuskoCSC) [inline docs/support/tutorials/gis/direct_allas.md:12]:
```suggestion
Reading data directly from an external service, like Allas or other object storage is slower than reading from local disks, for example reading a ~500 Mb file from your scratch directory on Puhti takes ~1 second, reading the same file from Allas takes ~10 seconds. In many cases, these seconds are negligible compared to the full duration of an analysis.
```

**Review by EetuHuuskoCSC — APPROVED** [review]
LGTM, one typo

**Comment by samumantha** [comment]
Please wait with merge for @ktiits approval :)

**Review by ktiits — APPROVED** [review]

**Comment by ktiits** [comment]
https://csc-guide-preview.rahtiapp.fi/origin/swi_vsicurl/support/tutorials/gis/gdal_cloud/ should now be ready for merging.

**Pull request merged**